### DISCUSSION TITLE
Upgrading to latest version of Agent v3.4.3.0

### DISCUSSION THREAD

**Discussion by rrroonn** [doitandbedone/ispyagentdvr-docker#240] [Q&A] [answered]
Hi,  
Firstly, thank you for providing and maintaining this image.

I have the lastest (:vlc) image installed and the web UI shows version v3.3.7.0.  I looked at your instructions re upgrading (though they just referenced libs) and followed the steps but as expected, the version remained the same.

If I want to update the latest version  v3.4.3.0, do you have a suggested approach? Or is there a reason that this container lags behind the current stable release of Agent?

thank you

Ron

**Comment by doitandbedone** [comment, marked as answer]
Been a bit busy lately, sorry for the delay! Just updated. Image should be
available in a few minutes!

**Reply by rrroonn** [reply]
Wow!  Thanks for doing this, and so quickly.
